code/modules/vending/magivend.dm

```diff
@@ -4,17 +4,17 @@
 	icon_state = "MagiVend"
 	product_slogans = "Sling spells the proper way with MagiVend!;Be your own Houdini! Use MagiVend!"
 	vend_reply = "Have an enchanted evening!"
-	product_ads = "FJKLFJSD;AJKFLBJAKL;1234 LOONIES LOL!;>MFW;Kill them fuckers!;GET DAT FUKKEN DISK;HONK!;EI NATH;Destroy the station!;Admin conspiracies since forever!;Space-time bending hardware!"
-	products = list(/obj/item/clothing/head/wizard = 1,
-		            /obj/item/clothing/suit/wizrobe = 1,
-		            /obj/item/clothing/head/wizard/red = 1,
-		            /obj/item/clothing/suit/wizrobe/red = 1,
-		            /obj/item/clothing/head/wizard/yellow = 1,
-		            /obj/item/clothing/suit/wizrobe/yellow = 1,
-		            /obj/item/clothing/shoes/sandal/magic = 1,
-		            /obj/item/staff = 2)
-	contraband = list(/obj/item/reagent_containers/glass/bottle/wizarditis = 1)	//No one can get to the machine to hack it anyways; for the lulz - Microwave
-	armor = list(MELEE = 100, BULLET = 100, LASER = 100, ENERGY = 100, BOMB = 0, BIO = 0, RAD = 0, FIRE = 100, ACID = 50)
+	product_ads = "EI NATH;Destroy the station!;Admin conspiracies since forever!;Space-time bending hardware!"
+	products = list(/obj/item/clothing/head/wizard = 3,
+		            /obj/item/clothing/suit/wizrobe = 3,
+		            /obj/item/clothing/head/wizard/red = 3,
+		            /obj/item/clothing/suit/wizrobe/red = 3,
+		            /obj/item/clothing/head/wizard/yellow = 3,
+		            /obj/item/clothing/suit/wizrobe/yellow = 3,
+		            /obj/item/clothing/shoes/sandal/magic = 3,
+		            /obj/item/staff = 3)
```

Comment:
add wands of nothing to this list

and maybe even staffs of healing, seeing as how you're giving them to mediborgs anyway